src/reconciler/fiberUpdater.js

```diff
@@ -18,12 +18,21 @@ export const updateLazy = (target, type) => {
         const C = resolveType(m.default)
         // chunks has been updated - new hot loader process is taking a place
         enterHotUpdate()
+        if (!React.forwardRef) {
```

Comment:
I thought this path was only used for `React.lazy` which is a 16.6 feature where React.forwardRef should be available. But react 15 uses this path as well which caused https://travis-ci.org/gaearon/react-hot-loader/jobs/516138894#L790